app/src/main/java/com/github/globant/githubsubscribers/subscriberslist/view/SubscribersAdapter.java

```diff
@@ -61,11 +65,27 @@ public SubscriberViewHolder(View itemView) {
             super(itemView);
             image = (ImageView) itemView.findViewById(R.id.img_avatar_subscriber);
             text = (TextView) itemView.findViewById(R.id.txt_login_subscriber);
+            itemView.setOnClickListener(this);
         }
 
         public void onBind(Subscriber item) {
             text.setText(item.getLogin());
             Picasso.with(image.getContext()).load(item.getAvataUrl()).into(image);
         }
+
+        public String getUserName() {
+            return text.getText().toString();
+        }
+
+        @Override
+        public void onClick(View view) {
+            if (clickListener != null) {
+                clickListener.onClickItemList(this, getAdapterPosition());
+            }
+        }
+    }
+
+    public interface ItemClickListener {
```

Comment:
In general, in our projects we have a rule to put interfaces for classes at the top, even above fields